spec/models/response_spec.rb

```diff
@@ -2,18 +2,18 @@
 
 RSpec.describe Response, type: :model do
   before do
-    skip unless ENV['DISABLE_USER_ANSWER'].present?
+    skip if ENV['DISABLE_USER_ANSWER'].blank?
   end
 
   describe 'dashboard' do
     let(:user) { create :user }
-    let(:question) {
+    let(:question) do
       # uncached
       # described_class.find_by(name: , training_module: { name: 'alpha' }).load.first
-  
+
       # cached
       Training::Module.by_name('alpha').page_by_name('1-1-4')
```

Comment:
Do still merge this but we will get a failure here if this were to run.

Reply:
It only runs if we have turned on responses (i.e. disabled user answer) so I held off until I had more time to understand the training changes (are they live?)